Source/WebCore/platform/cocoa/LocalizedStringsCocoa.mm

```diff
@@ -52,6 +52,11 @@ String contextMenuItemTagAddHighlightToNewQuickNote()
 }
 #endif
 
+String contextMenuItemTagCopyLinkToHighlight()
+{
+    return WEB_UI_STRING("Copy Link To Highlight", "Copy link to highlight context menu item");
```

Comment:
No sure if the "To" should be capitalized.

Reply:
I don't think it should be capitalized. Similarly, "Add to Quick Note" is not.